<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>Reports on Release Documentation - v1.0.3</title>
    <link>https://fantastic-fiesta-da4ab8e5.pages.github.io/reports.html</link>
    <description>Recent content in Reports on Release Documentation - v1.0.3</description>
    <generator>Hugo -- gohugo.io</generator>
    <language>en-us</language><atom:link href="https://fantastic-fiesta-da4ab8e5.pages.github.io/reports/index.xml" rel="self" type="application/rss+xml" />
    <item>
      <title>Code Coverage Test</title>
      <link>https://fantastic-fiesta-da4ab8e5.pages.github.io/reports/CodeCoverage-8ef78fba-102b-26d0-2963-2d641d1ec8bb.html</link>
      <pubDate>Mon, 01 Jan 0001 00:00:00 +0000</pubDate>
      
      <guid>https://fantastic-fiesta-da4ab8e5.pages.github.io/reports/CodeCoverage-8ef78fba-102b-26d0-2963-2d641d1ec8bb.html</guid>
      <description>Code Coverage Test    the code coverage is 0
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98 99 100 101 102 103 104 105 106 107 108 109 110 111 112 113 114 115 116 117 118 119 120  &amp;lt;?</description>
    </item>
    
    <item>
      <title>Unit Test Results</title>
      <link>https://fantastic-fiesta-da4ab8e5.pages.github.io/reports/UnitTest-5e450264-2228-1182-4c9b-74bae0231426.html</link>
      <pubDate>Mon, 01 Jan 0001 00:00:00 +0000</pubDate>
      
      <guid>https://fantastic-fiesta-da4ab8e5.pages.github.io/reports/UnitTest-5e450264-2228-1182-4c9b-74bae0231426.html</guid>
      <description>Test results for commit &amp;lsquo;a22d59b0fc7a6c63c9715a0dca9593261ca4d216&amp;rsquo; run at 0
   Tests Failures Errors     2 0 0    Results of test suite pytest    The following test where executed in test suite pytest
 NameExecution TimeResult test_for_get_speed0.009passed test_for_publish_to_topic0.002passed  Original Output    1  &amp;lt;?xml version=&amp;#34;1.0&amp;#34; encoding=&amp;#34;utf-8&amp;#34;?&amp;gt;&amp;lt;testsuites&amp;gt;&amp;lt;testsuite name=&amp;#34;pytest&amp;#34; errors=&amp;#34;0&amp;#34; failures=&amp;#34;0&amp;#34; skipped=&amp;#34;0&amp;#34; tests=&amp;#34;2&amp;#34; time=&amp;#34;0.236&amp;#34; timestamp=&amp;#34;2023-11-30T10:47:42.027555&amp;#34; hostname=&amp;#34;d667d091073d&amp;#34;&amp;gt;&amp;lt;testcase classname=&amp;#34;test_run&amp;#34; name=&amp;#34;test_for_get_speed&amp;#34; file=&amp;#34;test_run.py&amp;#34; line=&amp;#34;27&amp;#34; time=&amp;#34;0.009&amp;#34; /&amp;gt;&amp;lt;testcase classname=&amp;#34;test_run&amp;#34; name=&amp;#34;test_for_publish_to_topic&amp;#34; file=&amp;#34;test_run.</description>
    </item>
    
    <item>
      <title>Unit Test Results</title>
      <link>https://fantastic-fiesta-da4ab8e5.pages.github.io/reports/UnitTest-e2503874-3684-dc23-26a2-71f54745f9fe.html</link>
      <pubDate>Mon, 01 Jan 0001 00:00:00 +0000</pubDate>
      
      <guid>https://fantastic-fiesta-da4ab8e5.pages.github.io/reports/UnitTest-e2503874-3684-dc23-26a2-71f54745f9fe.html</guid>
      <description>Test results for commit &amp;lsquo;a22d59b0fc7a6c63c9715a0dca9593261ca4d216&amp;rsquo; run at 0
   Tests Failures Errors     1 0 0    Results of test suite pytest    The following test where executed in test suite pytest
 NameExecution TimeResult test_get_current_speed0.007passed  Original Output    1  &amp;lt;?xml version=&amp;#34;1.0&amp;#34; encoding=&amp;#34;utf-8&amp;#34;?&amp;gt;&amp;lt;testsuites&amp;gt;&amp;lt;testsuite name=&amp;#34;pytest&amp;#34; errors=&amp;#34;0&amp;#34; failures=&amp;#34;0&amp;#34; skipped=&amp;#34;0&amp;#34; tests=&amp;#34;1&amp;#34; time=&amp;#34;0.196&amp;#34; timestamp=&amp;#34;2023-11-30T10:56:21.714327&amp;#34; hostname=&amp;#34;fv-az530-677&amp;#34;&amp;gt;&amp;lt;testcase classname=&amp;#34;app.tests.integration.integration_test&amp;#34; name=&amp;#34;test_get_current_speed&amp;#34; file=&amp;#34;app/tests/integration/integration_test.py&amp;#34; line=&amp;#34;24&amp;#34; time=&amp;#34;0.007&amp;#34; /&amp;gt;&amp;lt;/testsuite&amp;gt;&amp;lt;/testsuites&amp;gt;   </description>
    </item>
    
    <item>
      <title>Vulnerability Scan Results</title>
      <link>https://fantastic-fiesta-da4ab8e5.pages.github.io/reports/vulnerability-scan-results.html</link>
      <pubDate>Mon, 01 Jan 0001 00:00:00 +0000</pubDate>
      
      <guid>https://fantastic-fiesta-da4ab8e5.pages.github.io/reports/vulnerability-scan-results.html</guid>
      <description>The scan was run on the container that was build using the code of the commit with the hash &amp;lsquo;a22d59b0fc7a6c63c9715a0dca9593261ca4d216&amp;rsquo; The number of vulnerabilities found was: &amp;lsquo;0&amp;rsquo;
Scan results for SampleApp-arch.tar (ubuntu 22.04)    The number of vulnerabilities in container SampleApp-arch.tar (ubuntu 22.04) is 0
Scan results for Python    The number of vulnerabilities in container Python is 1
Original Output    1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18  &amp;lt;?</description>
    </item>
    
  </channel>
</rss>
